app/javascript/react/assets/styles/Typography/Typography.css

```diff
@@ -0,0 +1,28 @@
+
+@font-face {
+    font-family: "Roboto";
+    font-weight: 400;
+    font-style: normal;
+    font-display: auto;
+    src: url("../../fonts/Roboto-Regular.ttf") format("ttf"),
```

Comment:
I think it would be better to use the fonts in woff format: https://fonts.google.com/knowledge/glossary/web_font